modules/programs/ssh/default.nix

```diff
@@ -97,28 +96,16 @@ let
 in
 
 {
+  imports = [
+    (mkRemovedOptionModule [ "services" "openssh" "authorizedKeysFiles" ] "No `nix-darwin` equivalent to this NixOS option.")
+  ];
```

Comment:
I added this because SSH keys specified in `~/.ssh/authorized_keys` stopped working (see #677), ideally it would be nice if this PR could maintain that, potentially by adding it to the `AuthorizedKeysCommand`

Reply:
I think that should work out of the box now, since we don’t override `AuthorizedKeysFile`:

              The  program should produce on standard output zero or more lines
              of authorized_keys output (see AUTHORIZED_KEYS in sshd(8)) .  Au‐
              thorizedKeysCommand is tried after the  usual  AuthorizedKeysFile
              files  and will not be executed if a matching key is found there.
              By default, no AuthorizedKeysCommand is run.